crates/hyperqueue/src/worker/bootstrap.rs

```diff
@@ -97,28 +97,29 @@ pub async fn initialize_worker(
             server_dir.access_filename()
         )
     })?;
-    let server_address = format!("{}:{}", record.worker.host, record.worker.port);
-    log::info!("Connecting to: {}", server_address);
 
     std::fs::create_dir_all(&configuration.work_dir)?;
     std::fs::create_dir_all(&configuration.log_dir)?;
```

Comment:
Found this by accident, but it seems to me like `log_dir` isn't actually used by anything in HQ?